tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/OpenTelemetry/StartRootSpanIntegration.cs

```diff
@@ -0,0 +1,77 @@
+// <copyright file="StartRootSpanIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.OpenTelemetry
+{
+    /// <summary>
+    /// Msmq calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "OpenTelemetry.Api",
+        TypeName = "OpenTelemetry.Trace.Tracer",
+        MethodName = "StartRootSpan",
+        ReturnTypeName = "OpenTelemetry.Trace.TelemetrySpan",
+        ParameterTypeNames = new[] { ClrNames.String, "OpenTelemetry.Trace.SpanKind", "OpenTelemetry.Trace.SpanAttributes", "System.Collections.Generic.IEnumerable`1[OpenTelemetry.Trace.Link]", "System.DateTimeOffset" },
+        MinimumVersion = "1.0.0",
+        MaximumVersion = "1.0.0",
+        IntegrationName = nameof(Configuration.IntegrationId.OpenTelemetry))]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class StartRootSpanIntegration
+    {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(StartSpanIntegration));
+
+        /// <summary>
+        /// OnMethodBegin callback
+        /// </summary>
+        /// <typeparam name="TTracer">Type of the tracer</typeparam>
+        /// <typeparam name="TSpanKind">Type of the span kind</typeparam>
+        /// <typeparam name="TSpanAttributes">Type of the span attributes</typeparam>
+        /// <typeparam name="TLinks">Type of the span links</typeparam>
+        /// <param name="instance">Instance value, aka `this` of the instrumented method.</param>
+        /// <param name="name">Span name.</param>
+        /// <param name="kind">Span kind.</param>
+        /// <param name="spanAttributes">Initial attributes for the span.</param>
+        /// <param name="links">Links for the span.</param>
+        /// <param name="startTimeinstance">Start time for the span.</param>
+        /// <returns>Calltarget state value</returns>
+        internal static CallTargetState OnMethodBegin<TTracer, TSpanKind, TSpanAttributes, TLinks>(TTracer instance, string name, TSpanKind kind, TSpanAttributes spanAttributes, TLinks links, DateTimeOffset startTimeinstance)
+        {
+            Log.Information("Entered OnMethodBegin for StartRootSpan");
```

Comment:
Isn't Info too high a level?

Reply:
This was leftover from debugging 😅 Deleted in c077562d5ce36031c100775b70977f1a88ca9f1a